usr/share/time-slider/lib/time_slider/zfs.py

```diff
@@ -235,7 +235,8 @@ def list_filesystems(self, pattern = None):
                                     (str(cmd), err, errdata)
             for line in outdata.rstrip().split('\n'):
                 line = line.rstrip().split()
-                Datasets.filesystems.append([line[0], line[1]])
+                if len(outdata) > 1:
```

Comment:
Shouldn't this appear before for loop?

Reply:
https://github.com/OpenIndiana/time-slider/pull/10 Thank you, good point.